cpplite/cpplite.editor/src/org/netbeans/modules/cpplite/editor/file/CDataObject.java

```diff
@@ -112,14 +116,27 @@
 @GrammarRegistration(grammar="resources/c.tmLanguage.json", mimeType=MIMETypes.C)
 public class CDataObject extends MultiDataObject {
 
+    static final String ICON = "org/netbeans/modules/cpplite/editor/file/resources/CSrcIcon.gif";
```

Comment:
Maybe, we can add `@StaticResource`.
Other files as well.

Reply:
Valid point - updated this accordingly.